components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/client/ApplicationManagementServiceClient.java

```diff
@@ -82,12 +83,13 @@ public ApplicationManagementServiceClient(String cookie, String backendServerURL
      * @param serviceProvider
      * @throws AxisFault
      */
-    public void createApplication(ServiceProvider serviceProvider) throws AxisFault {
+    public void createApplication(ServiceProvider serviceProvider, String templateName) throws AxisFault {
```

Comment:
Does not this make an API change ?

Reply:
Since this method will not get executed in client side, have removed it. Included a new method "createApplicationWithTemplate(ServiceProvider serviceProvider, String templateName) " instead of that.